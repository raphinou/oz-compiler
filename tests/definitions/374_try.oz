% try-finally expressions
% Exception raised
{Show try raise stopHere end b
      catch
         stopHere then
         'stopped'
      finally
         {Show 'and finished'}
      end}

% Success
{Show try allOk
      catch
         stopHere then
         'stopped'
      finally
         {Show 'and finished'}
      end}
% Success with integer value
{Show try 3
      catch
         stopHere then
         'stopped'
      finally
         {Show 'and finished'}
      end}

