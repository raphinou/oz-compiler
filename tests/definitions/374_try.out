'and finished'
stopped
'and finished'
allOk
'and finished'
3
